subsys/ipc/Kconfig

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 Nordic Semiconductor (ASA)
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig IPC_SERVICE
+	bool "IPC service for multiple users"
+	select BOARD_KEEP_SHM if BOARD_NRF5340PDK_NRF5340_CPUAPP || BOARD_NRF5340PDK_NRF5340_CPUAPPNS || BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPPNS
+	select IPM
+	select IPM_NRFX
```

Comment:
ditto, this makes it vendor specific and will cause issues if someone else enables this service.

Reply:
ditto